Header header
bool solution_completed
uint32 solution_status
uint32 position_type
int32 num_sats
